libcxx/CMakeLists.txt

```diff
@@ -218,6 +218,9 @@ set(LIBCXX_ABI_NAMESPACE "__${LIBCXX_ABI_VERSION}" CACHE STRING "The inline ABI
 if (NOT LIBCXX_ABI_NAMESPACE MATCHES "__.*")
   message(FATAL_ERROR "LIBCXX_ABI_NAMESPACE must be a reserved identifier, got '${LIBCXX_ABI_NAMESPACE}'.")
 endif()
+set(LIBCXX_AVAILABILITY_MINIMUM_HEADER_VERSION "1" CACHE STRING
```

Comment:
Is there a way that we can embed the version of libc++ that a program was built with into that program? I'm pretty sure it's possible to do that on some platforms with linker tricks. I think that would be useful.